web3_proxy/tests/common/app.rs

```diff
@@ -244,7 +248,7 @@ impl TestApp {
         // TODO: test influx
         // TODO: test redis
         let app_config: AppConfig = serde_json::from_value(json!({
-            "chain_id": 31337,
+            "chain_id": chain_id,
```

Comment:
other test files use 31337 hardcoded

Reply:
this is TestApp. It takes the chain_id from the other places